src/core/math/mat3.js

```diff
@@ -177,20 +177,34 @@ class Mat3 {
     /**
      * Generates the transpose of the specified 3x3 matrix.
      *
+     * @param {Mat3} [src] - The matrix to transpose. If not set, the current matrix is used.
```

Comment:
Do you think something like this might read better:

```
The matrix to transpose. If not set, the matrix is transposed in-place.
```

Reply:
Yep that sounds better. I've updated all instances thanks!